modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/jaxrs/exception/mapper/DuplicateObjectDefinitionExternalReferenceCodeExceptionMapper.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.rest.internal.jaxrs.exception.mapper;
+
+import com.liferay.object.exception.DuplicateObjectDefinitionExternalReferenceCodeException;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.vulcan.accept.language.AcceptLanguage;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
+
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Artur Souza
+ */
+@Component(
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Object.Admin.REST)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Object.Admin.REST.DuplicateObjectDefinitionExternalReferenceCodeExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+public class DuplicateObjectDefinitionExternalReferenceCodeExceptionMapper
+	extends BaseExceptionMapper
+		<DuplicateObjectDefinitionExternalReferenceCodeException> {
+
+	@Override
+	protected Problem getProblem(
+		DuplicateObjectDefinitionExternalReferenceCodeException
+			duplicateObjectDefinitionExternalReferenceCodeException) {
+
+		return new Problem(
+			Response.Status.BAD_REQUEST,
+			_language.get(
+				_acceptLanguage.getPreferredLocale(),
+				"This ERC is already in use. Try another one."));
```

Comment:
Hey @Artartur 

We need to add a language key here. I looked in `Language.properties` the key:
`please-enter-a-unique-external-reference-code=Please enter a unique external reference code.`

Is this the expect message? I not found in the LPS.